### ISSUE TITLE
Contact form-7-datepicker is not readable using JAWS (screen reader users)

### ISSUE THREAD

**Issue by ganesh-udmale** [relu/contact-form-7-datepicker#221]
Accessibility issue related to JAWS:

We noticed the following issues on the date picker widget: 
- When we tab through the dates of the datepicker calendar Jaws is not read selected day. it should read the selected date.
- Screen reader users not accessible next-prev button.
